linux_os/guide/services/ntp/chronyd_or_ntpd_specify_remote_server/rule.yml

```diff
@@ -44,7 +44,7 @@ references:
     cis: "3.6"
     cui: 3.3.7
     disa: "160,1891"
-    nist: AU-8(1),AU-8(1)(a)
+    nist: AU-8(1),AU-8(1)(a),AU-12(1)
```

Comment:
```suggestion
    nist: AU-8(1)(a)
```

Reply:
AU-8 (1)(a) | Compares the internal information system clocks [Assignment: organization-defined frequency] with [Assignment: organization-defined authoritative time source]; and
-- | --

Reply:
https://nvd.nist.gov/800-53/Rev4/control/AU-8#enhancement-1